webextension/backgroundscript.js

```diff
@@ -36,6 +36,6 @@ genericBrowser.browserAction.onClicked.addListener(handleClick);
 
 function handleClick(event) {
     genericBrowser.tabs.create({
-        url: genericBrowser.runtime.getURL('/index.html')
+        url: genericBrowser.runtime.getURL('/www/index.html')
```

Comment:
> It was to avoid adding the /index.html in the packages, which simply redirects to www/index.html.

It would have been nice to have this explanation in the commit message :)